pre_commit_hooks/file_contents_sorter.py

```diff
@@ -18,19 +18,18 @@
 
 
 def sort_file_contents(f):
-    before = tuple(f)
+    before = [line.strip(b'\n\r') for line in f if line.strip()]
```

Comment:
I also think (to be consistent) even if it does not need to perform any sorting it should still be removing blank lines and ensuring the file ends in a newline -- that way one could run this hook and then not expect *unexpected* changes to their files later